# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1498769087 14400
#      Thu Jun 29 16:44:47 2017 -0400
# Node ID d3a9ffc9fdbd3a0645f310a19cd8d962fc44991c
# Parent  69a111259a2c8030f71aadca772ee545b57e953e
move config macros from defaults.cc to a separate file

* default-defs.in.h: New file.  Move substituted macro definitions here
from defaults.in.cc.

* defaults.cc: Rename from defaults.in.cc.  Include default-defs.h.

* libinterp/corefcn/module.mk, libinterp/module.mk: Update.

diff --git a/libinterp/corefcn/default-defs.in.h b/libinterp/corefcn/default-defs.in.h
new file mode 100644
--- /dev/null
+++ b/libinterp/corefcn/default-defs.in.h
@@ -0,0 +1,181 @@
+// %NO_EDIT_WARNING%
+/*
+
+Copyright (C) 2017 John W. Eaton
+
+This file is part of Octave.
+
+Octave is free software; you can redistribute it and/or modify it
+under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 3 of the License, or
+(at your option) any later version.
+
+Octave is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Octave; see the file COPYING.  If not, see
+<http://www.gnu.org/licenses/>.
+
+*/
+
+// These are used by functions declared in defaults.h and defined in
+// defaults.cc.
+
+#if ! defined (OCTAVE_CANONICAL_HOST_TYPE)
+#  define OCTAVE_CANONICAL_HOST_TYPE %OCTAVE_CANONICAL_HOST_TYPE%
+#endif
+
+#if ! defined (OCTAVE_RELEASE)
+#  define OCTAVE_RELEASE %OCTAVE_RELEASE%
+#endif
+
+#if ! defined (OCTAVE_DEFAULT_PAGER)
+#  define OCTAVE_DEFAULT_PAGER %OCTAVE_DEFAULT_PAGER%
+#endif
+
+#if ! defined (OCTAVE_PREFIX)
+#  define OCTAVE_PREFIX %OCTAVE_PREFIX%
+#endif
+
+#if ! defined (OCTAVE_EXEC_PREFIX)
+#  define OCTAVE_EXEC_PREFIX %OCTAVE_EXEC_PREFIX%
+#endif
+
+#if ! defined (OCTAVE_BINDIR)
+#  define OCTAVE_BINDIR %OCTAVE_BINDIR%
+#endif
+
+#if ! defined (OCTAVE_DATADIR)
+#  define OCTAVE_DATADIR %OCTAVE_DATADIR%
+#endif
+
+#if ! defined (OCTAVE_DATAROOTDIR)
+#  define OCTAVE_DATAROOTDIR %OCTAVE_DATAROOTDIR%
+#endif
+
+#if ! defined (OCTAVE_INCLUDEDIR)
+#  define OCTAVE_INCLUDEDIR %OCTAVE_INCLUDEDIR%
+#endif
+
+#if ! defined (OCTAVE_LIBDIR)
+#  define OCTAVE_LIBDIR %OCTAVE_LIBDIR%
+#endif
+
+#if ! defined (OCTAVE_LIBEXECDIR)
+#  define OCTAVE_LIBEXECDIR %OCTAVE_LIBEXECDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALVERARCHLIBDIR)
+#  define OCTAVE_LOCALVERARCHLIBDIR %OCTAVE_LOCALVERARCHLIBDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALAPIARCHLIBDIR)
+#  define OCTAVE_LOCALAPIARCHLIBDIR %OCTAVE_LOCALAPIARCHLIBDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALARCHLIBDIR)
+#  define OCTAVE_LOCALARCHLIBDIR %OCTAVE_LOCALARCHLIBDIR%
+#endif
+
+#if ! defined (OCTAVE_ARCHLIBDIR)
+#  define OCTAVE_ARCHLIBDIR %OCTAVE_ARCHLIBDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALVEROCTFILEDIR)
+#  define OCTAVE_LOCALVEROCTFILEDIR %OCTAVE_LOCALVEROCTFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALAPIOCTFILEDIR)
+#  define OCTAVE_LOCALAPIOCTFILEDIR %OCTAVE_LOCALAPIOCTFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALOCTFILEDIR)
+#  define OCTAVE_LOCALOCTFILEDIR %OCTAVE_LOCALOCTFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_OCTFILEDIR)
+#  define OCTAVE_OCTFILEDIR %OCTAVE_OCTFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALVERFCNFILEDIR)
+#  define OCTAVE_LOCALVERFCNFILEDIR %OCTAVE_LOCALVERFCNFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALAPIFCNFILEDIR)
+#  define OCTAVE_LOCALAPIFCNFILEDIR %OCTAVE_LOCALAPIFCNFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALFCNFILEDIR)
+#  define OCTAVE_LOCALFCNFILEDIR %OCTAVE_LOCALFCNFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_FCNFILEDIR)
+#  define OCTAVE_FCNFILEDIR %OCTAVE_FCNFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_OCTDATADIR)
+#  define OCTAVE_OCTDATADIR %OCTAVE_OCTDATADIR%
+#endif
+
+#if ! defined (OCTAVE_OCTETCDIR)
+#  define OCTAVE_OCTETCDIR %OCTAVE_OCTETCDIR%
+#endif
+
+#if ! defined (OCTAVE_OCTINCLUDEDIR)
+#  define OCTAVE_OCTINCLUDEDIR %OCTAVE_OCTINCLUDEDIR%
+#endif
+
+#if ! defined (OCTAVE_OCTLIBDIR)
+#  define OCTAVE_OCTLIBDIR %OCTAVE_OCTLIBDIR%
+#endif
+
+#if ! defined (OCTAVE_OCTLOCALEDIR)
+#  define OCTAVE_OCTLOCALEDIR %OCTAVE_OCTLOCALEDIR%
+#endif
+
+#if ! defined (OCTAVE_OCTTESTSDIR)
+#  define OCTAVE_OCTTESTSDIR %OCTAVE_OCTTESTSDIR%
+#endif
+
+#if ! defined (OCTAVE_INFODIR)
+#  define OCTAVE_INFODIR %OCTAVE_INFODIR%
+#endif
+
+#if ! defined (OCTAVE_MANDIR)
+#  define OCTAVE_MANDIR %OCTAVE_MANDIR%
+#endif
+
+#if ! defined (OCTAVE_MAN1DIR)
+#  define OCTAVE_MAN1DIR %OCTAVE_MAN1DIR%
+#endif
+
+#if ! defined (OCTAVE_MAN1EXT)
+#  define OCTAVE_MAN1EXT %OCTAVE_MAN1EXT%
+#endif
+
+#if ! defined (OCTAVE_IMAGEDIR)
+#  define OCTAVE_IMAGEDIR %OCTAVE_IMAGEDIR%
+#endif
+
+#if ! defined (OCTAVE_LOCALSTARTUPFILEDIR)
+#  define OCTAVE_LOCALSTARTUPFILEDIR %OCTAVE_LOCALSTARTUPFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_STARTUPFILEDIR)
+#  define OCTAVE_STARTUPFILEDIR %OCTAVE_STARTUPFILEDIR%
+#endif
+
+#if ! defined (OCTAVE_INFOFILE)
+#  define OCTAVE_INFOFILE %OCTAVE_INFOFILE%
+#endif
+
+#if ! defined (OCTAVE_TEXI_MACROS_FILE)
+#  define OCTAVE_TEXI_MACROS_FILE %OCTAVE_TEXI_MACROS_FILE%
+#endif
+
+#if ! defined (OCTAVE_DOC_CACHE_FILE)
+#  define OCTAVE_DOC_CACHE_FILE %OCTAVE_DOC_CACHE_FILE%
+#endif
diff --git a/libinterp/corefcn/defaults.in.cc b/libinterp/corefcn/defaults.cc
rename from libinterp/corefcn/defaults.in.cc
rename to libinterp/corefcn/defaults.cc
--- a/libinterp/corefcn/defaults.in.cc
+++ b/libinterp/corefcn/defaults.cc
@@ -1,9 +1,8 @@
-// DO NOT EDIT!  Generated automatically by subst-default-vals.
 /*
 
 Copyright (C) 1996-2017 John W. Eaton
 
 This file is part of Octave.
 
 Octave is free software; you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by
@@ -48,178 +47,17 @@ along with Octave; see the file COPYING.
 #include "ovl.h"
 #include "ov.h"
 #include "parse.h"
 #include "interpreter.h"
 #include "unwind-prot.h"
 #include "variables.h"
 #include "version.h"
 
-// These are used by functions declared in defaults.h and defined in
-// defaults.cc.
-
-#if ! defined (OCTAVE_CANONICAL_HOST_TYPE)
-#  define OCTAVE_CANONICAL_HOST_TYPE %OCTAVE_CANONICAL_HOST_TYPE%
-#endif
-
-#if ! defined (OCTAVE_RELEASE)
-#  define OCTAVE_RELEASE %OCTAVE_RELEASE%
-#endif
-
-#if ! defined (OCTAVE_DEFAULT_PAGER)
-#  define OCTAVE_DEFAULT_PAGER %OCTAVE_DEFAULT_PAGER%
-#endif
-
-#if ! defined (OCTAVE_PREFIX)
-#  define OCTAVE_PREFIX %OCTAVE_PREFIX%
-#endif
-
-#if ! defined (OCTAVE_EXEC_PREFIX)
-#  define OCTAVE_EXEC_PREFIX %OCTAVE_EXEC_PREFIX%
-#endif
-
-#if ! defined (OCTAVE_BINDIR)
-#  define OCTAVE_BINDIR %OCTAVE_BINDIR%
-#endif
-
-#if ! defined (OCTAVE_DATADIR)
-#  define OCTAVE_DATADIR %OCTAVE_DATADIR%
-#endif
-
-#if ! defined (OCTAVE_DATAROOTDIR)
-#  define OCTAVE_DATAROOTDIR %OCTAVE_DATAROOTDIR%
-#endif
-
-#if ! defined (OCTAVE_INCLUDEDIR)
-#  define OCTAVE_INCLUDEDIR %OCTAVE_INCLUDEDIR%
-#endif
-
-#if ! defined (OCTAVE_LIBDIR)
-#  define OCTAVE_LIBDIR %OCTAVE_LIBDIR%
-#endif
-
-#if ! defined (OCTAVE_LIBEXECDIR)
-#  define OCTAVE_LIBEXECDIR %OCTAVE_LIBEXECDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALVERARCHLIBDIR)
-#  define OCTAVE_LOCALVERARCHLIBDIR %OCTAVE_LOCALVERARCHLIBDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALAPIARCHLIBDIR)
-#  define OCTAVE_LOCALAPIARCHLIBDIR %OCTAVE_LOCALAPIARCHLIBDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALARCHLIBDIR)
-#  define OCTAVE_LOCALARCHLIBDIR %OCTAVE_LOCALARCHLIBDIR%
-#endif
-
-#if ! defined (OCTAVE_ARCHLIBDIR)
-#  define OCTAVE_ARCHLIBDIR %OCTAVE_ARCHLIBDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALVEROCTFILEDIR)
-#  define OCTAVE_LOCALVEROCTFILEDIR %OCTAVE_LOCALVEROCTFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALAPIOCTFILEDIR)
-#  define OCTAVE_LOCALAPIOCTFILEDIR %OCTAVE_LOCALAPIOCTFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALOCTFILEDIR)
-#  define OCTAVE_LOCALOCTFILEDIR %OCTAVE_LOCALOCTFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_OCTFILEDIR)
-#  define OCTAVE_OCTFILEDIR %OCTAVE_OCTFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALVERFCNFILEDIR)
-#  define OCTAVE_LOCALVERFCNFILEDIR %OCTAVE_LOCALVERFCNFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALAPIFCNFILEDIR)
-#  define OCTAVE_LOCALAPIFCNFILEDIR %OCTAVE_LOCALAPIFCNFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALFCNFILEDIR)
-#  define OCTAVE_LOCALFCNFILEDIR %OCTAVE_LOCALFCNFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_FCNFILEDIR)
-#  define OCTAVE_FCNFILEDIR %OCTAVE_FCNFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_OCTDATADIR)
-#  define OCTAVE_OCTDATADIR %OCTAVE_OCTDATADIR%
-#endif
-
-#if ! defined (OCTAVE_OCTETCDIR)
-#  define OCTAVE_OCTETCDIR %OCTAVE_OCTETCDIR%
-#endif
-
-#if ! defined (OCTAVE_OCTINCLUDEDIR)
-#  define OCTAVE_OCTINCLUDEDIR %OCTAVE_OCTINCLUDEDIR%
-#endif
-
-#if ! defined (OCTAVE_OCTLIBDIR)
-#  define OCTAVE_OCTLIBDIR %OCTAVE_OCTLIBDIR%
-#endif
-
-#if ! defined (OCTAVE_OCTLOCALEDIR)
-#  define OCTAVE_OCTLOCALEDIR %OCTAVE_OCTLOCALEDIR%
-#endif
-
-#if ! defined (OCTAVE_OCTTESTSDIR)
-#  define OCTAVE_OCTTESTSDIR %OCTAVE_OCTTESTSDIR%
-#endif
-
-#if ! defined (OCTAVE_INFODIR)
-#  define OCTAVE_INFODIR %OCTAVE_INFODIR%
-#endif
-
-#if ! defined (OCTAVE_MANDIR)
-#  define OCTAVE_MANDIR %OCTAVE_MANDIR%
-#endif
-
-#if ! defined (OCTAVE_MAN1DIR)
-#  define OCTAVE_MAN1DIR %OCTAVE_MAN1DIR%
-#endif
-
-#if ! defined (OCTAVE_MAN1EXT)
-#  define OCTAVE_MAN1EXT %OCTAVE_MAN1EXT%
-#endif
-
-#if ! defined (OCTAVE_IMAGEDIR)
-#  define OCTAVE_IMAGEDIR %OCTAVE_IMAGEDIR%
-#endif
-
-#if ! defined (OCTAVE_LOCALSTARTUPFILEDIR)
-#  define OCTAVE_LOCALSTARTUPFILEDIR %OCTAVE_LOCALSTARTUPFILEDIR%
-#endif
-
-#if ! defined (OCTAVE_STARTUPFILEDIR)
-#  define OCTAVE_STARTUPFILEDIR %OCTAVE_STARTUPFILEDIR%
-#endif
-
-// These are used by variables defined in other files.  Maybe they
-// should be moved?  Or maybe it is best to have just one generated file
-// containing variable substitutions?
-
-#if ! defined (OCTAVE_INFOFILE)
-#  define OCTAVE_INFOFILE %OCTAVE_INFOFILE%
-#endif
-
-#if ! defined (OCTAVE_TEXI_MACROS_FILE)
-#  define OCTAVE_TEXI_MACROS_FILE %OCTAVE_TEXI_MACROS_FILE%
-#endif
-
-#if ! defined (OCTAVE_DOC_CACHE_FILE)
-#  define OCTAVE_DOC_CACHE_FILE %OCTAVE_DOC_CACHE_FILE%
-#endif
+#include "default-defs.h"
 
 static bool initialized = false;
 
 static std::string Voctave_home;
 static std::string Voctave_exec_home;
 
 static std::string Vbin_dir;
 static std::string Vdata_dir;
diff --git a/libinterp/corefcn/module.mk b/libinterp/corefcn/module.mk
--- a/libinterp/corefcn/module.mk
+++ b/libinterp/corefcn/module.mk
@@ -125,16 +125,17 @@ COREFCN_SRC = \
   %reldir%/coct-hdf5-types.c \
   %reldir%/comment-list.cc \
   %reldir%/conv2.cc \
   %reldir%/daspk.cc \
   %reldir%/dasrt.cc \
   %reldir%/dassl.cc \
   %reldir%/data.cc \
   %reldir%/debug.cc \
+  %reldir%/defaults.cc \
   %reldir%/defun.cc \
   %reldir%/det.cc \
   %reldir%/dirfns.cc \
   %reldir%/display.cc \
   %reldir%/dlmread.cc \
   %reldir%/dot.cc \
   %reldir%/dynamic-ld.cc \
   %reldir%/eig.cc \
@@ -246,17 +247,17 @@ COREFCN_SRC = \
   %reldir%/xdiv.cc \
   %reldir%/xnorm.cc \
   %reldir%/xpow.cc \
   %reldir%/zfstream.cc \
   $(NOINSTALL_COREFCN_INC)
 
 ## Special rules for sources which must be built before rest of compilation.
 
-%reldir%/defaults.cc: %reldir%/defaults.in.cc build-aux/subst-default-vals.sh | %reldir%/$(octave_dirstamp)
+%reldir%/default-defs.h: %reldir%/default-defs.in.h build-aux/subst-default-vals.sh | %reldir%/$(octave_dirstamp)
 	$(AM_V_GEN)$(call simple-filter-rule,build-aux/subst-default-vals.sh)
 
 %reldir%/graphics.h: %reldir%/graphics.in.h %reldir%/genprops.awk | %reldir%/$(octave_dirstamp)
 	$(AM_V_GEN)rm -f $@-t && \
 	$(AWK) -f $(srcdir)/%reldir%/genprops.awk $< > $@-t && \
 	mv $@-t $@
 
 %reldir%/graphics-props.cc: %reldir%/graphics.in.h %reldir%/genprops.awk | %reldir%/$(octave_dirstamp)
@@ -288,16 +289,19 @@ COREFCN_SRC = \
 	$(AWK) 'BEGIN { print "// DO NOT EDIT. AUTOMATICALLY GENERATED FROM oct-tex-symbols.in."; print "static uint32_t symbol_codes[][2] = {"; count = 0; } END { print "};"; printf("static int num_symbol_codes = %d;\n", count); } !/^#/ && (NF == 3) { printf("  { %s, %s },\n", $$2, $$3); count++; }' $< > $@-t && \
 	mv $@-t $@
 
 %reldir%/oct-tex-lexer.cc: LEX_OUTPUT_ROOT := lex.octave_tex_
 
 %reldir%/oct-tex-parser.yy: %reldir%/oct-tex-parser.in.yy
 	$(AM_V_GEN)$(call subst-bison-api-decls,octave_tex_)
 
+noinst_HEADERS += \
+  %reldir%/default-defs.h
+
 noinst_LTLIBRARIES += \
   %reldir%/libcorefcn.la
 
 %canon_reldir%_libcorefcn_la_SOURCES = $(COREFCN_SRC)
 
 %canon_reldir%_libcorefcn_la_CPPFLAGS = \
   $(libinterp_liboctinterp_la_CPPFLAGS) \
   $(FFTW_XCPPFLAGS) \
@@ -307,17 +311,17 @@ noinst_LTLIBRARIES += \
   $(LLVM_CPPFLAGS) \
   $(Z_CPPFLAGS)
 
 %canon_reldir%_libcorefcn_la_CFLAGS = $(AM_CFLAGS) $(WARN_CFLAGS)
 
 %canon_reldir%_libcorefcn_la_CXXFLAGS = $(AM_CXXFLAGS) $(WARN_CXXFLAGS) $(LLVM_CXXFLAGS)
 
 libinterp_EXTRA_DIST += \
-  %reldir%/defaults.in.cc \
+  %reldir%/default-defs.in.h \
   %reldir%/genprops.awk \
   %reldir%/graphics.in.h \
   %reldir%/mk-errno-list.sh \
   %reldir%/mk-mxarray-h.in.sh \
   %reldir%/mxarray.in.h \
   %reldir%/oct-errno.in.cc \
   %reldir%/oct-tex-lexer.in.ll \
   %reldir%/oct-tex-parser.in.yy \
diff --git a/libinterp/module.mk b/libinterp/module.mk
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -25,17 +25,17 @@
 %canon_reldir%_liboctinterp_la_CFLAGS = $(AM_CFLAGS) $(WARN_CFLAGS)
 
 %canon_reldir%_liboctinterp_la_CXXFLAGS = $(AM_CXXFLAGS) $(WARN_CXXFLAGS)
 
 octlib_LTLIBRARIES += %reldir%/liboctinterp.la
 
 BUILT_SOURCES += \
   %reldir%/builtin-defun-decls.h \
-  %reldir%/corefcn/defaults.cc \
+  %reldir%/corefcn/default-defs.h \
   %reldir%/corefcn/graphics-props.cc \
   %reldir%/corefcn/graphics.h \
   %reldir%/corefcn/mxarray.h \
   %reldir%/corefcn/oct-tex-parser.h \
   %reldir%/corefcn/oct-tex-symbols.cc \
   %reldir%/parse-tree/oct-gperf.h \
   %reldir%/parse-tree/oct-parse.h \
   %reldir%/version.h
@@ -65,17 +65,17 @@ LIBINTERP_BUILT_DISTFILES = \
 ## Files that are created during build process and installed,
 ## BUT not distributed in tarball.
 LIBINTERP_BUILT_NODISTFILES = \
   %reldir%/build-env.cc \
   %reldir%/build-env-features.cc \
   %reldir%/liboctinterp-build-info.cc \
   %reldir%/corefcn/mxarray.h \
   %reldir%/corefcn/oct-errno.cc \
-  %reldir%/corefcn/defaults.cc \
+  %reldir%/corefcn/default-defs.h \
   %reldir%/corefcn/graphics-props.cc \
   %reldir%/corefcn/graphics.h \
   %reldir%/builtin-defun-decls.h \
   %reldir%/operators/ops.cc \
   %reldir%/version.h \
   %reldir%/builtins.cc
 
 %canon_reldir%_EXTRA_DIST += \
@@ -145,17 +145,17 @@ endif
   %reldir%/octave.cc \
   %reldir%/version.cc \
   $(LIBINTERP_OPERATORS_SRC) \
   $(TEMPLATE_INST_SRC)
 
 nodist_%canon_reldir%_liboctinterp_la_SOURCES = \
   %reldir%/corefcn/mxarray.h \
   %reldir%/corefcn/oct-errno.cc \
-  %reldir%/corefcn/defaults.cc \
+  %reldir%/corefcn/default-defs.h \
   %reldir%/corefcn/graphics.h \
   %reldir%/operators/ops.cc \
   %reldir%/builtin-defun-decls.h \
   %reldir%/builtins.cc \
   %reldir%/build-env.cc \
   %reldir%/build-env-features.cc \
   %reldir%/liboctinterp-build-info.cc \
   %reldir%/version.h
@@ -179,17 +179,16 @@ nodist_%canon_reldir%_liboctinterp_la_SO
   -version-info $(%canon_reldir%_liboctinterp_version_info) \
   $(NO_UNDEFINED_LDFLAG) \
   -bindir $(bindir) \
   $(LIBOCTINTERP_LINK_OPTS) \
   $(WARN_LDFLAGS)
 
 ULT_DIST_SRC := \
   $(filter-out $(GENERATED_PARSER_FILES), $(DIST_SRC)) \
-  %reldir%/corefcn/defaults.in.cc \
   $(ULT_PARSER_SRC)
 
 LIBINTERP_FOUND_DEFUN_FILES := \
   $(shell $(SHELL) $(srcdir)/build-aux/find-defun-files.sh "$(srcdir)" $(ULT_DIST_SRC))
 
 BUILT_IN_DEFUN_FILES = $(OPT_HANDLERS) $(LIBINTERP_FOUND_DEFUN_FILES)
 
 DLDFCN_DEFUN_FILES = $(DLDFCN_SRC)
